daemon/indexing/apollo/index.js

```diff
@@ -255,7 +255,12 @@ const resolvers = {
       return listing.category
     },
     price(listing) {
-      return {currency: 'ETH', amount: listing.priceEth}
+      if(listing.priceCurrency === undefined
```

Comment:
The schema enforces that both of those are set and therefore origin-js marketplace.getListing should never return a Listing object that does not have those set. But agreed better to be paranoid and have those checks at different layer of the stack.

Reply:
Given that origin.js now guarantees us a title, I'll remove this check.

Reply:
Also apollo schema guarantees us that

Reply:
Well the Apollo schema crashes that part of the request if it's empty, which is what I was trying to avoid.

Reply:
Aha yes you are right... the `origin-js` guarantees the object validity at indexing time while apollo schema only at query time.

Reply:
Aha yes you are right... the `origin-js` guarantees the object validity at indexing time while apollo schema only at query time.